<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
    background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(34, 148, 157, 255), stop:0.890052 rgba(255, 177, 177, 255));
}
</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="main_layout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="organ_layout">
        <property name="spacing">
         <number>40</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
    color: #FFFFFF; /* Couleur du texte */
    font: 15pt &quot;Ubuntu&quot;;
    padding: 5px; /* Ajustement de la marge intérieure */
    border: 1px solid rgba(0, 25, 12, 0.4); /* Couleur de la bordure */
    border-radius: 5px; /* Arrondi des coins */
    background-color: transparent; /* Couleur de fond */
    selection-background-color: rgba(34, 148, 157, 1); /* Couleur de fond de la sélection */

	padding-top: 1px; /* Ajustez la marge supérieure pour centrer verticalement le texte */
    padding-bottom: 1px;
}

QComboBox:hover {
    border-color: #FFFFFF; /* Couleur de la bordure lorsque survolé */
}

QComboBox::drop-down {
    border: none; /* Suppression de la flèche de liste déroulante par défaut */
}


QComboBox QAbstractItemView {
    border: 2px solid darkgray; /* Bordure de la liste déroulante */
    background-color:rgba(69, 154, 160, 0.7); 
    selection-background-color: rgba(34, 148, 157, 1); /* Couleur de fond de la sélection */
}</string>
          </property>
          <item>
           <property name="text">
            <string>Liver</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right kidney</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Spleen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pancreas</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Aorta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Inferior vena cava</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right adrenal gland</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left adrenal gland</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Gallbladder</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Esophagus</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stomach</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Duodenum</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Left kidney</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 3px;
}

QSlider::sub-page:horizontal {
    border: 1px solid #999999;
    height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: rgba(0, 25, 12, 0.4);
    margin: 2px 0;
}</string>
          </property>
          <property name="sliderPosition">
           <number>99</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="glass_button">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QToolButton {
	border-radius: 15px;
    background-color: transparent;
    border: 1px solid transparent;
    color: #FFFFFF; /* Couleur du texte */
}

QToolButton:hover {
    background-color: rgba(0, 0, 0, 0.2); /* Couleur de fond lorsqu'il est survolé */
}</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>glass.png</normaloff>glass.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>35</width>
            <height>35</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="volume_button">
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    color: #FFFFFF; /* Couleur du texte */
	font: 15pt &quot;Ubuntu&quot;;
}

QCheckBox::indicator {
	border-radius: 10px;
	padding: 1.5px;
    background-color: rgba(255, 255, 255, 1); /* Couleur de fond de la case à cocher */
    border: 1px solid #FFFFFF; /* Couleur de la bordure de la case à cocher */
    width: 15px; /* Largeur de la case à cocher */
    height: 15px; /* Hauteur de la case à cocher */
}

QCheckBox::indicator:hover {
    background-color: rgba(0, 0, 0, 0.2); /* Couleur de fond de la case à cocher lorsqu'elle est survolée */
}

QCheckBox::indicator:checked {
    background-color: rgba(0, 255, 0, 0.4); /* Couleur de fond de la case à cocher lorsqu'elle est cochée */
    border-color: #FFFFFF;
}

</string>
        </property>
        <property name="text">
         <string>Volume rendering</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="label_button">
        <property name="styleSheet">
         <string notr="true">QCheckBox {
    color: #FFFFFF; /* Couleur du texte */
	font: 15pt &quot;Ubuntu&quot;;
}

QCheckBox::indicator {
	border-radius: 10px;
	padding: 1.5px;
    background-color: rgba(255, 255, 255, 1); /* Couleur de fond de la case à cocher */
    border: 1px solid #FFFFFF; /* Couleur de la bordure de la case à cocher */
    width: 15px; /* Largeur de la case à cocher */
    height: 15px; /* Hauteur de la case à cocher */
}

QCheckBox::indicator:hover {
    background-color: rgba(0, 0, 0, 0.2); /* Couleur de fond de la case à cocher lorsqu'elle est survolée */
}

QCheckBox::indicator:checked {
    background-color: rgba(0, 255, 0, 0.4); /* Couleur de fond de la case à cocher lorsqu'elle est cochée */
    border-color: #FFFFFF;
}

</string>
        </property>
        <property name="text">
         <string>Label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="ruler_button">
        <property name="styleSheet">
         <string notr="true">QToolButton {
	font: 15pt &quot;Ubuntu&quot;;
    background-color: transparent;
    border: 1px solid transparent;
    color: #FFFFFF; /* Couleur du texte */
}

QToolButton:hover {
    background-color: rgba(0, 0, 0, 0.2); /* Couleur de fond lorsqu'il est survolé */
}

QToolButton:pressed {
    background-color: rgba(0, 0, 0, 0.4); /* Couleur de fond lorsqu'il est cliqué */
}</string>
        </property>
        <property name="text">
         <string>Add ruler</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="image_layout">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>792</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
